conjur/controller/user_controller.py

```diff
@@ -35,7 +35,7 @@ def rotate_api_key(self):
         try:
             # pylint: disable=line-too-long
             resource_to_update, new_api_key = self.user_logic.rotate_api_key(self.user_input_data.user_id)
-            sys.stdout.write(f"Successfully rotated API key for '{resource_to_update}'. "
+            sys.stdout.write(f"Successfully rotated API key for '{resource_to_update}'.\n "
```

Comment:
space seems redundant now:
```suggestion
            sys.stdout.write(f"Successfully rotated API key for '{resource_to_update}'.\n"
```